app/assets/javascripts/discourse/app/components/hide-modal-trigger.js

```diff
@@ -1,7 +0,0 @@
-import Component from "@ember/component";
-export default Component.extend({
-  didInsertElement() {
-    this._super(...arguments);
-    $(".d-modal.fixed-modal").modal("hide").addClass("hidden");
```

Comment:
This was running after the modal had been removed from the DOM... so I think it was a no-op 🤷